cdap-watchdog/src/test/java/io/cdap/cdap/logging/clean/LogCleanerTest.java

```diff
@@ -107,29 +120,153 @@ public static void cleanUp() {
   }
 
   @Test
-  public void testLogCleanup() throws Exception {
+  @Parameters({
```

Comment:
It's a dep that's not developer for last 5 years. It's better not to introduce it

Reply:
This is just a test dependency used for adding parameterized tests. Helpful in increasing test coverage.
Other alternative would be to initialize an Object[][] for test cases which is not very readable.